gfx.h

```diff
@@ -48,7 +48,8 @@ enum GfxCreateContextFlag
 };
 typedef uint32_t GfxCreateContextFlags;
 
-GfxContext gfxCreateContext(HWND window, GfxCreateContextFlags flags = 0, IDXGIAdapter *adapter = nullptr);
+GfxContext gfxCreateContext(HWND window, GfxCreateContextFlags flags, IDXGIAdapter *adapter = nullptr);
+GfxContext gfxCreateContext(uint32_t window_width, uint32_t window_height, GfxCreateContextFlags flags, IDXGIAdapter *adapter = nullptr);
```

Comment:
Any reason for removing the default value on initialization flags? `GfxCreateContextFlags flags = 0`.
The lib's examples now fail to build: https://github.com/gboisse/gfx/tree/master/examples

Reply:
Well the default value was removed because otherwise compiler could not distinguish between overloads in case two arguments are provided. Unfortunately I didn't know examples existed; can we just modify them to accomodate to these changes?

Reply:
The way it's done in https://github.com/gboisse/gfx/pull/123 works fin though.